src/test/java/org/takes/rs/RsTextTest.java

```diff
@@ -58,4 +59,22 @@ public void makesPlainTextResponse() throws IOException {
         );
     }
 
+    /**
+     * RsText can build a response with a status.
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void makesTextResponseWithStatus() throws IOException {
+        MatcherAssert.assertThat(
+            new RsPrint(
+                new RsText(
+                    new RsWithStatus(HttpURLConnection.HTTP_NOT_FOUND),
+                    "something not found"
+                )
+            ).printHead(),
```

Comment:
@mbarbieri can't we get the status code and assert it instead of getting the head text?